modules/importing-manifest-list-through-imagestreamimport.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+// * assembly/openshift_images/managing-image-streams.adoc
+
+:_content-type: PROCEDURE
+[id="importing-manifest-list-through-imagestreamimport_{context}"]
+= Importing a manifest list through ImageStreamImport
+
+Use the following procedure to import a manifest list through the `ImageStreamImport` object with the `importMode` value. 
+
+.Procedure
+
+. Create an `ImageStreamImport` object that sets the `importMode` to `Manifestlist` by running the following command:
++
+[source,terminal]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: image.openshift.io/v1
+kind: ImageStreamImport
+metadata:
+  name: app
+  namespace: <namespace>
+spec:
+  import: true
+  images:
+  - from:
+      kind: DockerImage
+      name: quay.io/<user_name>/<image_name>
+    to:
+      name: latest
+    referencePolicy:
+      type: Source
+    importPolicy:
+      importMode: "ManifestList"
+EOF
+----
+
+. Enter the following command to ensure that Image objects were created for the manifest list: 
++
+[source,terminal]
+----
+$ oc get image | grep quay.io/<user_name>/<image_name>
+----
+
+. Enter the following command to ensure that all sub-manifests were created:
++
+[source,terminal]
+----
+$ oc get is app
+----
++
+[NOTE]
+====
+You can compare the output from the aforementioned commands against the content of `quay.io/<user_name>/<image_name>`. If correct, there should be one image for the manifest list, and one image for each sub-manifest. 
+====
+
+. (Optiona) Delete the images previously created by entering the following command:
++
+[source,terminal]
+----
+$ oc delete images $(oc get images | grep quay.io/<user_name>/<image_name> | awk '{ print $1 }')
+----
+
+. (Optional) delete the ImageStream by entering the following command:
```

Comment:
If the image stream existed before these steps and had some other images, this command will delete them too. Perhaps we don't need to tell how to undo previous steps.

Reply:
Yes, let's keep it simple and remove optional cleanup steps.

We may want to explain what is outcome of creating ImageStreamImport, but today we don't have anything like this, even for normal imports. AFAIK we don't have any docs about ImageStreamImport.